docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_YOLO_From_Tensorflow.md

```diff
@@ -144,9 +144,22 @@ cd darkflow
 
 To convert YOLOv1 or YOLOv2 model to TensorFlow, go to the root directory of the cloned DarkFlow repository and run the following command:<br>
 ```sh
-python3 ./flow --model <path_to_model>/<model_name>.cfg --load <path_to_model>/<model_name>.weights --savepb
+python3 flow --model <path_to_model>/<model_name>.cfg --load <path_to_model>/<model_name>.weights --labels <path_to_dataset_labels_file> --savepb
 ```
 
+For Yolo V1 `--labels` argument can be skipped:
```

Comment:
Possibly better to describe cases like yolov1 and yolov2 with voc and then describe command line for other cases (if any).

Reply:
you mean firstly show particular cases and only after that general command?

because currently both of version use VOC and export command for both of them is given but files for v1 does not contain that in the name while v2 contain.

Reply:
F.e. by the link for Yolo V1 https://pjreddie.com/darknet/yolov1/ it's written that it's trained on Pascal VOC 

![image](https://user-images.githubusercontent.com/5703039/123089277-694ae700-d42f-11eb-93c8-eb0e50531da6.png)

but name does not contain `VOC` postfix

![image](https://user-images.githubusercontent.com/5703039/123089111-330d6780-d42f-11eb-887a-6d8be87e94f2.png)

Reply:
Yes, right I mean to describe particular cases, and after that common if there are any other cases than 2 described already. I think user can try to run common command and then find out that something should be changed for Yolo v2.

Reply:
put v1 conversion command first and general command last. Please review